Model/Transaction/Refund.php

```diff
@@ -77,9 +77,26 @@ public function build(
         );
 
         if (isset($this->request['line_items'])) {
```

Comment:
@ryanreeves-taxjar Is it possible for a refund to only have an adjustment with no line items refunded? I imagine in this scenario we'd simply pass the refund-level amounts and this code would be fine as-is. Just checking.

Reply:
Confirmed that this works for credit memos with only adjustments, no items.